test/e2e/tests/network/network-connection.spec.ts

```diff
@@ -63,9 +68,7 @@ networkConfigs.forEach((config) => {
             .withPermissionControllerConnectedToTestDapp()
             .withEnabledNetworks({
               eip155: {
-                [CHAIN_IDS.MONAD_TESTNET]: true,
-                [CHAIN_IDS.MEGAETH_TESTNET]: true,
-                [CHAIN_IDS.SEI]: true,
+                [config.chainId]: true,
```

Comment:
Update test state to better reflect real world state - we only show 1 network custom network/testnet at a time.